sootup.core/src/main/java/sootup/core/util/printer/JimplePrinter.java

```diff
@@ -196,8 +191,12 @@ public void printTo(SootClass<?> cl, PrintWriter out, LabeledStmtPrinter printer
           SootField f = (SootField) fieldIt.next();
           printer.newline();
           printer.handleIndent();
-          printer.literal(f.getDeclaration());
```

Comment:
do we need/use/want .getDeclaration() somewhere else? maybe we can remove it.. I mean its nice to have when needed - but I would not expect to find it and its basically just concatenating two SootField methods which I would do quickly myself before searching for getDeclaration() unless I stumbled over it already.